edx_api_doc_tools/apps.py

```diff
@@ -10,7 +13,11 @@
 
 class EdxApiDocToolsConfig(AppConfig):
     """
-    Configuration for the edx_api_doc_tools Django application.
-    """
+    Configuration for this app.
 
+    # TODO: How do we make it so 'drf_yasg' is automatically
+    # installed as a Django app when 'edx_api_doc_tools'
+    # is added to INSTALLED_APPS? Currently, both must be added manually.
+    """
```

Comment:
It's probably not a great idea to automatically add a line to INSTALLED_APPS, as there might be multiple applications which add this app. Also, at what position would that application be added?
I don't think it's necessary to automatically add the app, it's not a big inconvenience for the developers.

Reply:
Fair; I was imagining that `INSTALLED_APPS` is more similar to a requirements.txt file (where duplicates are fine and order doesn't matter) than it actually is.

As a workaround, I've added a check to `EdxApiDocToolsConfig.ready` that will raise `ImproperlyConfigured` if drf-yasg isn't listed.